.github/workflows/tag.yml

```diff
@@ -0,0 +1,37 @@
+name: Update website on push
+
+on:
+  push:
+    tags:
+    - *
+
+jobs:
+  build-deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout homebrew-puppet
+      uses: actions/checkout@v1
+      with:
+        repository: puppetlabs/homebrew-puppet
+        ref: master
+    - name: Update version and SHA
+      run: |
+        curl -OL https://github.com/puppetlabs/wash/archive/${GITHUB_REF}.tar.gz
+        sha=$(sha256sum ${GITHUB_REF}.tar.gz)
+        sed -e "s/version = \".*\"/version = \"${GITHUB_REF}\"/g" -i '' Formula/wash.rb
+        sed -e "s/sha256 \".*\"/sha256 \"${sha}\"/g" -i '' Formula/wash.rb
+
+        git config user.name "${GITHUB_ACTOR}"
+        git config user.email "${GITHUB_ACTOR}@users.noreply.github.com"
+        publish_branch="wash-${GITHUB_REF}"
+        git checkout -b ${publish_branch}
+        commit_message="Automated update: $(date -u) ${GITHUB_REF}"
+        git commit -am ${commit_message}
+
+        git push origin ${publish_branch}
+        PULLS_URI="https://api.github.com/repos/puppetlabs/homebrew-puppet/pulls"
+        AUTH_HEADER="Authorization: token $GITHUB_TOKEN"
+        new_pr_resp=$(curl --data {"title":"${commit_message}","head":"${publish_branch}","base":"master"} -X POST -s -H "${AUTH_HEADER}" ${PULLS_URI})
+        echo "$new_pr_resp"
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I have no idea if this token has permission to push a new branch to puppetlabs/homebrew-puppet. Probably have to test this out somehow. Maybe by temporarily changing it to `on.push.branch: master`.

Reply:
I think I'll need to generate a token that has access to homebrew-puppet and add it as a secret on this repository.